model_zoo/models/alexnet/model_alexnet.prototext

```diff
@@ -1,5 +1,4 @@
 model {
-  name: "directed_acyclic_graph_model"
```

Comment:
Why deleting this? Is "directed_acyclic_graph_model" default model name in LBANN?

Reply:
I've changed DAG models to be the default.